views/01UCS_LAL-PRIMO_REDESIGN/css/ucla/account.css

```diff
@@ -67,3 +67,30 @@ md-list-item p.text-headings {
   color: var(--color-black);
   font-size: 20px !important;
 }
+
+/* First column of fine information (labels) */
```

Comment:
@kjallen To build on what @ztucker4 said, we've been using a comment convention like this:
`/* Fine information (first column: labels):  U/Body/Overline */ `
And then removing the redundant comment pasted in with the actual CSS from the design spec.

This will (theoretically) make it easier to find & change things, if the `Body/Overline` design style changes.

Reply:
Corrected comment structure to reflect `U/*` format